app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  has_one :user
+
+  def update_post_counter(name)
+    @user = User.find_by(name: name)
+    @user.postsCounter = Post.where(author_id: @user.id).count
+    @user.save
+  end
+
+  def get_comments(post)
+    @post = Post.find_by(id: post)
+    @comments = Comment.where(post_id: @post.id).limit(5)
+  end
```

Comment:
- [ ] Please update these methods with the below to work for it as expected. ( `after_save` will be called every time a post is created for a user so it will increment the post counter).

```ruby
 after_save :update_posts_counter

  def update_posts_counter
    author.increment!(:posts_counter)
  end

  def five_recent_comments
    comments.order(created_at: :desc).limit(5)
  end